app/models/application_letter.rb

```diff
@@ -46,7 +45,7 @@ def after_deadline?
   # @param none
   # @return [Boolean] true if no status changes are allowed anymore
   def status_change_allowed?
-    !event.application_status_locked
+    !event.participant_selection_locked
```

Comment:
I think this function should be changed soon, atm it allows changing the status in both draft and application phase (which is also the reason, why some of our tests are still working that might break otherwise). However, this is not part of this PR, just wanted to point it out